utils/checks/validators/format.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import os
+
+from . import (ChangelogValidator, LogoValidator, ManifestValidator,
+               MetaValidator, ParserValidator, TaxonomyValidator,
+               TestFileValidator)
+from .constants import CheckResult
+
+
+class FormatValidator:
+    def __init__(
+        self, path: str, module_result: CheckResult, args: argparse.Namespace
+    ) -> None:
+        self.path = path
+
+        format_name = os.path.basename(path)
+        self.result = CheckResult(
+            name=f"check_format_{format_name}",
+            description="Checks the format has a proper definition",
+            options={
+                "path": path,
+                "module_result": module_result,
+                "module_path": module_result.options["path"],
+            },
+        )
+
+        self.validators = [
+            MetaValidator,
+            LogoValidator,
+            ManifestValidator,
```

Comment:
For the format, the ManifestValidator doesn't check the existence of a description and it doesn't check the data_sources too.